Assets/Scripts/UI/DialogBox/Settings/DialogBoxSettings.cs

```diff
@@ -78,7 +78,7 @@ public void OnClickSave()
         this.CloseDialog();
         WorldController.Instance.spawnInventoryController.SetUIVisibility(developerModeToggle.isOn);
         LocalizationTable.SetLocalization(languageDropdown.value);
-        //FurnitureBuildMenu.instance.RebuildMenuButtons(developerModeToggle.isOn);
+        ///FurnitureBuildMenu.instance.RebuildMenuButtons(developerModeToggle.isOn);
```

Comment:
This works for StyleCop (was only fixing a warning), I don't think that it needs changed. At some point we will need to fix that error and call a similar function.